data/frequencies/grab_wortschatz_data.py

```diff
@@ -67,4 +69,10 @@ def main() -> None:
     logging.basicConfig(
         format="%(filename)s %(levelname)s: %(message)s", level="INFO"
     )
-    main()
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument(
+        "--freq-json-path",
+        default=WORTSCHATZ_DICT_PATH,
+        help="Path to the JSON file for the Wortschatz frequency download URLs",
```

Comment:
Nit: should be lowercase `path to...`  to match the style of autogenerated flags like for `--help` etc.